tests/test_synthia_format.py

```diff
@@ -0,0 +1,58 @@
+
+from unittest import TestCase
+import os.path as osp
+
+import numpy as np
+
+from datumaro.components.annotation import Mask
+from datumaro.components.dataset import Dataset
+from datumaro.components.extractor import DatasetItem
+from datumaro.util.test_utils import compare_datasets
+import datumaro.plugins.synthia_format as Synthia
+
+from .requirements import Requirements, mark_requirement
+
+DUMMY_DATASET_DIR = osp.join(osp.dirname(__file__), 'assets',
+    'synthia_dataset')
+
+class SynthiaImporterTest(TestCase):
+    @mark_requirement(Requirements.DATUM_497)
+    def test_can_import(self):
```

Comment:
How about a test that uses a `labels.txt` file?

Reply:
Added test with custom label map.